### DISCUSSION TITLE
General

### DISCUSSION THREAD

**Discussion by shahzadhaider1** [kashifkhan0771/utils#47] [General]
General discussions regarding anything related to Go or this repository.

**Comment by shahzadhaider1** [comment]
@kashifkhan0771 What versioning strategy are we using for this repository?

**Comment by kashifkhan0771** [comment]
@shahzadhaider1 Currently, we are in the development phase (`v0.x.y`). Once issue [#41](https://github.com/kashifkhan0771/utils/issues/41) is resolved, I’ll draft the first stable release following semantic versioning (`v1.x.y`).  

From that point forward:
- Any **Major** changes will increment the `v1` to `v2`.  
- **Minor** updates, such as new features, will increment the `x`.  
- **Patch** updates, like bug fixes, will increment the `y`.

**Comment by kashifkhan0771** [comment]
Library views jumped to 200+ on 13th January 2025 🥳
